components/brave_wallet/browser/solana_tx_manager_unittest.cc

```diff
@@ -407,7 +411,8 @@ class SolanaTxManagerUnitTest : public testing::Test {
   }
 
  protected:
-  base::test::TaskEnvironment task_environment_;
+  content::BrowserTaskEnvironment task_environment_;
```

Comment:
Also don't need `content` here and in other new appearances if component stays content-free

Reply:
fixed in 6505b8dd6397978a8667a4ca135829a53a256b44